tests/smoke/base.sh

```diff
@@ -55,6 +55,14 @@ elif command -v maturin &> /dev/null; then
   pip install -e .
 elif [ "$(pip list|grep -c scikit_build_core)" -ne "0" ]; then
   pip install -e .
+elif [ "$(pip list|grep -c pybind11)" -ne "0" ]; then
+  # Assuming you are inside the root of the CMake source directory
+  mkdir build
```

Comment:
ideally you should run just `pip install -e .` and it should run cmake under the hook